templates/city.html

```diff
@@ -112,7 +110,33 @@ <h2 class="course-city-heading">Aktuální kurzy a srazy</h2>
     </div>
     {% endif %}
   </section>
+  {% elif not current_meetups and city_slug != 'ostatni' %}
+  <section class=" container text-center" id="meetups">
+    <h2 class="course-city-heading">Nedávné kurzy a srazy</h2>
+    <div class="meetup">
+      {% for meetup in past_meetups %}
+        {% if loop.index < 4 %}
+         {{ meetup_info(meetup,  with_registration_link=False) }}
+        {% endif %}
+      {% endfor %}
+    </div>
+    <div>
+        <a href="#past-meetups">
+            {% if past_meetups|length == 0 %}
+                Kurz se připravuje.
+            {% elif past_meetups|length == 1 %}
+                Jedna další akce už je
+            {% elif past_meetups|length < 5 %}
+                A další {{past_meetups|length}} akce už jsou
+            {% else %}
+                A dalších {{past_meetups|length-3}} kurzů a srazů už je
+            {% endif %}
+            za námi!
```

Comment:
Tady to "za námi" se ukáže i za tím "Kurz se připravuje."
Bylo by lepší nakopírovat "za námi" za každou hlášku, která to potřebuje.